docs/sources/tools/logcli.md

```diff
@@ -250,26 +250,22 @@ usage: logcli query [<flags>] <query>
 
 Run a LogQL query.
 
-The "query" command is useful for querying for logs. Logs can be returned in a
-few output modes:
+The "query" command is useful for querying for logs. Logs can be returned in a few output modes:
 
   raw: log line
   default: log timestamp + log labels + log line
   jsonl: JSON response from Loki API of log line
 
-The output of the log can be specified with the "-o" flag, for example, "-o raw"
-for the raw output format.
+The output of the log can be specified with the "-o" flag, for example, "-o raw" for the raw output format.
 
-The "query" command will output extra information about the query and its
-results, such as the API URL, set of common labels, and set of excluded labels.
-This extra information can be suppressed with the --quiet flag.
+The "query" command will output extra information about the query and its results, such as the API URL, set of common labels, and set of
+excluded labels. This extra information can be suppressed with the --quiet flag.
 
-By default we look over the last hour of data; use --since to modify or provide
-specific start and end times with --from and --to respectively.
+By default we look over the last hour of data; use --since to modify or provide specific start and end times with --from and --to
+respectively.
 
-Notice that when using --from and --to then ensure to use RFC3339Nano time
-format, but without timezone at the end. The local timezone will be added
-automatically or if using --timezone flag.
+Notice that when using --from and --to then ensure to use RFC3339Nano time format, but without timezone at the end. The local timezone will
```

Comment:
```suggestion
Note that when using --from and --to you must use the RFC3339Nano time format, but without timezone at the end. The local timezone will
```

Reply:
@JStickler Hi, what you are commenting on here is generated, it's not changes that I made. It is output of the help text of the application. A few are from new stuff that I added and I will change that, do you want me to change all of these things? Maybe it would be better to make a PR yourself to change it? I feel it's a bit out of the scope of my contribution here. IDK what do you think?